examples/cms-batch/gen.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import itertools
+import random
+import sys
+
+FEATURES = ["few", "small", "equal", "max"]
+
+
+def gen(input_name: str) -> None:
+    max_n = [10, 10**5]["few" not in input_name]
+    max_k = [10**5, 10**9]["small" not in input_name]
```

Comment:
Python supports a variant of [the ternary operator](https://docs.python.org/3/reference/expressions.html#conditional-expressions), making hacks like this unneeded.